rclrs/src/parameter.rs

```diff
@@ -1366,7 +1369,7 @@ mod tests {
     #[test]
     fn test_optional_parameter_apis() {
         let ctx = Context::new([]).unwrap();
-        let node = create_node(&ctx, "param_test_node").unwrap();
+        let node = create_node(&ctx, &format!("param_test_node_{}", line!())).unwrap();
```

Comment:
Are those changes to make sure we don't have name clashes when running tests?

Reply:
Yep, the tests were producing lots of spam warnings because of clashing node names while tests ran in parallel.

Reply:
For some extra context, pre-Jazzy, the rcl node was responsible for creating the rosout publisher and the rosout logging code would warn if it had already created a publisher for a node with the same name.
In the Jazzy (and rolling) releases, the creation of the rosout publisher is no longer the responsibility of the rcl node and so the issue doesn't occur.
It does also mean that we will need to have Jazzy (and on) specific code for explicitly creating the rosout publisher.  Following the rclcpp/rclpy implementations, this can easily be added to the node builder code.  I'm happy to create a ticket for this so the work doesn't get lost if that helps?